app/2.0/docs/tools/create-app-polymer-cli.md

```diff
@@ -0,0 +1,124 @@
+architecture](https://developers.google.com/web/updates/2015/11/app-shell) pattern.
+
+There are fundamental concepts of the app shell architecture that you should understand before 
+creating your app project with Polymer CLI: the entrypoint, the shell, and fragments. See [App 
+structure](../../toolbox/prpl#app-structure) from the App Toolbox docs for an in-depth overview of 
+these concepts.
+
+## Set up basic app project {#basic-app}
+
+Follow the steps below to get your `basic` app project set up.
+
+1.  Create a directory for your app project.
+
+    <pre><code>mkdir <var>app</var>
+    cd <var>app</var></code></pre>
+
+    Where <code><var>app</var></code> is the name of your project directory.
+
+1.  Initialize your app. Polymer CLI asks you a few questions
+    as it sets up your app.
+
+        polymer init
+
+1.  Select `polymer-2-application`.
+
+1.  Enter a name for your app. Defaults to the name of the current directory.
+
+1.  Enter a name for the main element in your project. The main element is the
+    top-most, application-level element of your app. Defaults to the name of
+    the current directory, followed by `-app`.
+
+    The code samples throughout this doc use the example app element name
+    <code><var>my-app</var></code>. When creating your app you'll want to
+    replace any instance of <code><var>my-app</var></code> with the name of
+    your main element.
+    {.alert .alert-info}
+
+1.  Enter a description for your app.
+
+The Polymer CLI generates your app and installs its dependencies.
+
+## App project layout
+
+After creating your app, Polymer CLI generates the following files and directories:
+
+*   `bower.json`. Configuration file for Bower.
+*   `bower_components/`. Project dependencies. See [Manage dependencies](#dependencies).
+*   `index.html`. Entrypoint page of the app.
+*   `src/`<code><var>my-app</var></code>/<code><var>my-app</var></code>`.html`.
+    Source code for main element.
+*   `test/`<code><var>my-app</var></code>/<code><var>my-app</var></code>`_test.html`. Tests 
+    for main element.
+
+### Add elements
+
+You may want to compose your main element out of smaller elements specific to your app. These 
+application-specific elements should be defined in the `src` directory, at the same level as 
+<code><var>my-app</var></code>.
+
+<code>
+    app/
+      src/
+        <var>my-app</var>/
+          <var>my-app</var>.html
+        <var>my-el</var>/
+          <var>my-el</var>.html
+</code>
+
+To add another element to the project:
+
+1. Create a new folder under src.
+   
+<code>
+   mkdir src/<var>my-el</var>
+</code>
+   
+2. Create an HTML import for the new element. You can use the existing app element as a starting point.
+   
+3. To use the new element, you'll need to import it into your application element (for example, <code><var>my-app</var>.html</code>) with an "import" link:
+   
+<code>
+   <link rel="import" href="/src/<var>my-el</var>/<var>my-el</var>.html">
```

Comment:
Again, indent four spaces. Plus, you'll need to replace the `<` and `>` with entities: `&gt;` in place of `<` and `&lt;` in place of `>` ... The markdown blocks escape these automatically. When you do a `<code>` block yourself, you need to escape them or this gets interpreted as a literal `<link>` tag and it is hidden from the user.